packages/dev/core/src/Shaders/iblPdf.fragment.fx

```diff
@@ -0,0 +1,48 @@
+precision highp sampler2D;
+precision highp samplerCube;
+
+#include<helperFunctions>
+
+#define PI 3.1415927
+varying vec2 vUV;
+
+#ifdef IBL_USE_CUBE_MAP
+uniform samplerCube iblSource;
+#else
+uniform sampler2D iblSource;
+#endif
+uniform sampler2D normalizationSampler;
+uniform int iblWidth;
+uniform int iblHeight;
+
+#ifdef IBL_USE_CUBE_MAP
+vec3 equirectangularToCubemapDirection(vec2 uv) {
```

Comment:
Perhaps factor this function and `fetchLuminance` into a `ShadersInclude/iblFunctions.fx` file, to avoid duplication?